src/adapters/swagger20.es6

```diff
@@ -104,24 +101,38 @@ function createTransaction(transition, method) {
 /*
  * Parse Swagger 2.0 into Refract elements
  */
-export function parse({ source }, done) {
+export function parse({minim, source}, done) {
   // TODO: Will refactor this once API Description namespace is stable
   // Leaving as large block of code until then
-  let basePath = source.basePath || '';
-  let schemaDefinitions = _.pick(source, 'definitions') || {};
+  const Asset = minim.getElementClass('asset');
+  const Copy = minim.getElementClass('copy');
+  const Category = minim.getElementClass('category');
+  const HrefVariables = minim.getElementClass('hrefVariables');
+  const HttpHeaders = minim.getElementClass('httpHeaders');
+  const MemberElement = minim.getElementClass('member');
+  const Resource = minim.getElementClass('resource');
+  const Transition = minim.getElementClass('transition');
+
+  const paramToElement = convertParameterToElement.bind(
+    convertParameterToElement, minim);
```

Comment:
It passes the linter checks, but I'm open to suggestions for `paramToElement`. For the lines above, they'll [eventually](https://github.com/refractproject/minim/blob/master/CHANGELOG.md) use destructuring as well via something like `const {Asset, Copy, Category} = minim.elements;` but that is pending a new upcoming Minim release.